Description: Remove JavaScript from Google related to Custom Search Engine queries
Author: Christopher Baines <mail@cbaines.net>

Last-Update: 2015-10-25

--- scikit-learn-0.16.1.orig/doc/themes/scikit-learn/layout.html
+++ scikit-learn-0.16.1/doc/themes/scikit-learn/layout.html
@@ -338,15 +338,4 @@
       })();
     </script>
     {% endif %}
-
-    <script src="http://www.google.com/jsapi" type="text/javascript"></script>
-    <script type="text/javascript"> google.load('search', '1',
-        {language : 'en'}); google.setOnLoadCallback(function() {
-            var customSearchControl = new
-            google.search.CustomSearchControl('016639176250731907682:tjtqbvtvij0');
-            customSearchControl.setResultSetSize(google.search.Search.FILTERED_CSE_RESULTSET);
-            var options = new google.search.DrawOptions();
-            options.setAutoComplete(true);
-            customSearchControl.draw('cse', options); }, true);
-    </script>
 {%- endblock %}
